src/manifest.json

```diff
@@ -10,10 +10,10 @@
         "48": "icon-48.png",
         "128": "icon-128.png"
     },
-    "incognito": "split",
     "background": {
         "service_worker": "background.js"
     },
+    "action": { },
```

Comment:
Is this empty object necessary?

Reply:
Oh that's just to make sure that the browser action is pinnable / clickable. Depending whether you're inclined to keep it, could switch it for

"action": {
"default_title": "Switch to / create Spotify tab."
},
for a little more explanation? (Would require an additional localisation message though).

Or just omit it entirely and focus solely on the keyboard shortcuts. I had originally wanted to use it for the 'queue-search' command until I realised chrome disallows you from setting the browser action as a global keyboard shortcut.

Reply:
Nah it's okay, let's keep it as-is.